[All Hosts] (ui) how to create add-in commands with unified manifest (#3947)
OfficeDev/office-js-docs-pr: docs/develop/create-addin-commands.md

---
title: Create add-in commands with the XML manifest
description: Configure an XML manifest to define add-in commands for Excel, Outlook, PowerPoint, and Word. Use add-in commands to create UI elements, add buttons or lists, and perform actions.
ms.date: 04/07/2023
ms.localizationpriority: medium
---

> # Create add-in commands with the XML manifest

Reviewer: Please check for links to this article and rename the text as appropriate.

Add-in commands provide an easy way to customize the default Office user interface (UI) with specified UI elements that perform actions. For an introduction to add-in commands, see [Add-in commands](../design/add-in-commands.md).

This article describes how to edit your XML manifest to define add-in commands and how to create the code for [function commands](../design/add-in-commands.md#types-of-add-in-commands). 

> [!TIP]

> > Instructions for creating add-in commands with the unified Microsoft 365 manifest are in [Create add-in commands with the unified Microsoft 365 manifest](create-addin-commands-unified-manifest.md).

Reviewer: ```suggestion
> For instructions on how to create add-in commands with the unified Microsoft 365 manifest, see [Create add-in commands with the unified Microsoft 365 manifest](create-addin-commands-unified-manifest.md).
```
